.github/workflows/images-legacy.yaml

```diff
@@ -0,0 +1,94 @@
+name: Image CI Build
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+jobs:
+  build-and-push-prs:
```

Comment:
Same as above, but for PRs on mirrored repositories. I believe we need an `if: github.repository == 'cilium/cilium'` statement here as well.